src/commands/project/deposit.ts

```diff
@@ -22,7 +23,7 @@ export default class ProjectDeposit extends TransactionCommand {
     const output = [];
     const { tx: approveTx, deadline, sig } = await approve(signer, usdc, projects, amount);
     if (approveTx) output.push(await run(approveTx, signer, [usdc]));
-    const tx = await projects.populateTransaction.depositProjectEscrow(id, amount, deadline, sig.v, sig.r, sig.s);
+    const tx = await projects.populateTransaction.depositProjectEscrow(signerAddress, id, amount, deadline, sig.v, sig.r, sig.s);
```

Comment:
can we add this param conditionally if `flags.network !== 'testnet-sepolia'`? that way we can merge this now and once we release onto testnet we can update the ABI and remove the conditional

Reply:
Updated accordingly